gremlin/templates/apparmor-loader.yaml

```diff
@@ -0,0 +1,77 @@
+{{ if .Values.gremlin.installApparmorProfile }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: apparmor-loader
+  # Namespace must match that of the ConfigMap.
+  namespace: {{ .Release.Namespace}}
+  labels:
+    daemon: apparmor-loader
+spec:
+  selector:
+    matchLabels:
+      daemon: apparmor-loader
+  template:
+    metadata:
+      name: apparmor-loader
+      labels:
+        daemon: apparmor-loader
+    spec:
+      initContainers:
+      - command:
+        - sh
+        - -c
+        - apparmor_parser -R /profiles/agent_apparmor.profile || true
+        image: google/apparmor-loader:latest
+        imagePullPolicy: IfNotPresent
+        name: clean-old-profiles
+        resources: {}
+        securityContext:
+          privileged: true
```

Comment:
Can this be replaced with the underlying capabilities the container needs?

Reply:
This is what the apparmor-loader container is documented as needing (it's not my container)

I could try to figure out a tighter set maybe? I dunno

Reply:
Unfortunately it looks like google's loader does require privileged.  There are alternate solutions but they rely on things like ssh access to the underlying nodes :(

Reply:
I'll take a look at this today while I work through testing this locally. My assumption here is that anything that's `privileged` can be replaced with an explicit set of capabilities needed to perform the work. I'll report back what I find.